examples/recovery/simple.json

```diff
@@ -15,7 +15,7 @@
       "is": "Ministry of Health Welfare and Sport",
       "df": "2021-06-12",
       "du": "2021-11-28",
-      "ci": "URN:UVCI:01:NL:DADFCC47C7334E45A906DB12FD859FB7"
+      "ci": "URN:UVCI:01:NL:DADFCC47C7334E45A906DB12FD859FB7#1"
```

Comment:
The checksum should only be part of the UVCI displayed on paper. And these examples are probably meant to be included in the DCC.

Reply:
We should add specific paper and dcc examples, but for now these should be as "complete" as we can make them (so using all of the features).